### PR TITLE
correct badly formatted HTML in TPN reports

### PR DESCRIPTION
**Pull request by nadinequinr3**
Process-related internal information [here](https://r3-cev.atlassian.net/wiki/spaces/TW/pages/4401332261/Generating+a+Snyk+License+Report#Correcting-Badly-Formatted-HTML).

### CODE REVIEW

**Review by NeilAndersonR3 — APPROVED** [review]
Preview looks good!

File: content/en/platform/corda/5.0/introduction/release-notes/legal-info/corda-cli.html [hunk 1]
```diff
@@ -992,8 +992,8 @@ <h1>
       </div>
       </div>
     </div>
-    </div>
-  </main>
+
+
 </main></body>
```
Comment (yarivus) [inline content/en/platform/corda/5.0/introduction/release-notes/legal-info/corda-cli.html:997]: I prefer this is fixed accordingly:
```
  </main>
</body>
```
(no prior newlines)

File: content/en/platform/corda/5.0/introduction/release-notes/legal-info/corda-gradle-plugins.html [hunk 2]
```diff
@@ -1296,8 +1296,7 @@ <h1>
       </div>
       </div>
     </div>
-    </div>
-  </main>
+
 </main></body>
```
Comment (yarivus) [inline content/en/platform/corda/5.0/introduction/release-notes/legal-info/corda-gradle-plugins.html:1300]: same here

**Review by yarivus — COMMENTED** [review]
see comments

**Comment by nadinequinr3** [comment]
Thanks, @yarivus. I'll bear these comments in mind if this comes up again. The files are, in general, very strangely formatted and so these lines are relatively very minor.

**Pull request merged**